drivers/unix/ip_unix.cpp

```diff
@@ -55,17 +55,13 @@
 #include <iphlpapi.h>
 #endif // MINGW hack
 #endif
-#else
+#else // UNIX
 #include <netdb.h>
-#ifdef ANDROID_ENABLED
-#include "platform/android/ifaddrs_android.h"
-#else
 #ifdef __FreeBSD__
 #include <sys/types.h>
 #endif
-#include <ifaddrs.h>
-#endif
 #include <arpa/inet.h>
+#include <ifaddrs.h>
```

Comment:
Well, seems like the Android NDK does have `ifaddrs.h`, but it's only usable once we have `__ANDROID_API__ >= 24`:
```
#if __ANDROID_API__ >= 24
void freeifaddrs(struct ifaddrs* __ptr) __INTRODUCED_IN(24);
int getifaddrs(struct ifaddrs** __list_ptr) __INTRODUCED_IN(24);
#endif /* __ANDROID_API__ >= 24 */
```
So I guess we'll have to keep the old file for now.